src/models/config.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * SPDX-FileCopyrightText: 2021 Nextcloud GmbH and Nextcloud contributors
```

Comment:
Please add the correct authors sth like
```suggestion
 * SPDX-FileLicenseText: 2024 STRATO AG
 * SPDX-License-Identifier: AGPL-3.0-or-later
 * SPDX-FileContributor: Misha M.-Kupriyanov <kupriyanov@strato.de>
```